google/cloud/aiplatform/_featurestores/entity_type.py

```diff
@@ -0,0 +1,368 @@
+
+        Example Usage:
+
+            my_entity_type = aiplatform.EntityType(
+                entity_type_name='my_entity_type_id',
+                featurestore_id='my_featurestore_id',
+            )
+            my_entityType.list_features()
+
+        Args:
+            filter (str):
+                Optional. Lists the Features that match the filter expression. The
+                following filters are supported:
+
+                -  ``value_type``: Supports = and != comparisons.
+                -  ``create_time``: Supports =, !=, <, >, >=, and <=
+                   comparisons. Values must be in RFC 3339 format.
+                -  ``update_time``: Supports =, !=, <, >, >=, and <=
+                   comparisons. Values must be in RFC 3339 format.
+                -  ``labels``: Supports key-value equality as well as key
+                   presence.
+
+                Examples:
+
+                -  ``value_type = DOUBLE`` --> Features whose type is
+                   DOUBLE.
+                -  ``create_time > \"2020-01-31T15:30:00.000000Z\" OR update_time > \"2020-01-31T15:30:00.000000Z\"``
+                   --> EntityTypes created or updated after
+                   2020-01-31T15:30:00.000000Z.
+                -  ``labels.active = yes AND labels.env = prod`` -->
+                   Features having both (active: yes) and (env: prod)
+                   labels.
+                -  ``labels.env: *`` --> Any Feature which has a label with
+                   'env' as the key.
+            order_by (str):
+                Optional. A comma-separated list of fields to order by, sorted in
+                ascending order. Use "desc" after a field name for
+                descending. Supported fields:
+
+                -  ``feature_id``
+                -  ``value_type``
+                -  ``create_time``
+                -  ``update_time``
+
+        Returns:
+            List[Features] - A list of managed feature resource objects.
+        """
+        return _featurestores.Feature.list(
+            entity_type_name=self.resource_name, filter=filter, order_by=order_by,
+        )
+
+    @base.optional_sync()
+    def delete_features(
+        self, feature_ids: List[str], sync: Optional[bool] = True,
+    ) -> None:
+        """Deletes feature resources in this EntityType given their feature IDs.
+        WARNING: This deletion is permanent.
+
+        Args:
+            feature_ids (List[str]):
+                Required. The list of feature IDs to be deleted.
+            sync (bool):
+                Optional. Whether to execute this deletion synchronously. If False, this method
+                will be executed in concurrent Future and any downstream object will
+                be immediately returned and synced when the Future has completed.
+        """
+        for feature_id in feature_ids:
+            feature = self.get_feature(feature_id=feature_id)
+            feature.delete(sync=sync)
+            if not sync:
```

Comment:
To ensure deletions are as concurrent as possible and reduce the time to delete all features, `wait` should be called after `delete` is invoked on all features. Currently features are deleted sequentially.